tracer/src/Datadog.Trace/Agent/Transports/HttpStreamRequest.cs

```diff
@@ -39,8 +38,8 @@ public void AddHeader(string name, string value)
             using (var bidirectionalStream = _streamFactory.GetBidirectionalStream())
             {
                 var content = new BufferContent(segment);
-                _headers.Add("Content-Type", contentType);
                 var request = new HttpRequest("POST", _uri.Host, _uri.PathAndQuery, _headers, content);
+                _headers.Add("Content-Type", contentType);
```

Comment:
Any reason for the move here, or just an artefact of changing the approach?

Reply:
Ah, no, just added it to line 42 because that's where it was previously, so a mental error on my part.